weblate/templates/accounts/snippets/login-info.html

```diff
@@ -22,10 +22,10 @@
 
 {% elif private_commit_email_opt_in %}
   <p>
-    {% translate "By registering you agree to use your name and e-mail in version control system commits and provide your contribution under license defined by each translated project." %}
+    {% translate "By registering, you agree that your name and username will be visible to other users signed in to this server as long as your account exists. Additionally, your name and e-mail can be used in version control system commits to author the changes you made; it can be stored there indefinitely. You can enable using a private commit e-mail in your profile. Your contribution will be published under the license defined by each project you contributed to." %}
```

Comment:
```suggestion
    {% translate "By registering, you agree that your name and username will be visible to other users signed in to this server as long as your account exists." %}
    {% translate "Additionally, your name and e-mail can be used in version control system commits to author the changes you made; it can be stored there indefinitely." %}
    {% translate "You can enable using a private commit e-mail in your profile." %}
    {% translate "Your contribution will be published under the license defined by each project you contributed to." %}
```